README.md

```diff
@@ -1,15 +1,90 @@
 # matchesWhereObject( object, whereQueryObject )
 
-Determines if `object` matches `whereQueryObject`, returning `true` or `false`.
+This function determines if the provided `object` matches every comparison clause in `whereQueryObject`, returning `true` or `false` accordingly.
 
 `whereQueryObject` is an object of the form:
 
+
 ```
 {
 	<attribute> : <value>
-	<attribute> : [ <value>, <value> ]
-	 ...
+	<attribute> : [ <value1>, ..., <valueN> ]
+	<attribute> : { comparator : <comparator>, value : <value> }
 }
 ```
 
-Each attribute is either a value or an array of values. If the attribute is a single value (primitive or object), object.attribute must strictly equal the value to result in a match. Otherwise, if teh attribute is is an array, only records for which object.attribute is strictly equaly to one of the elements in the array will result in a match.
\ No newline at end of file
+# Comparisons
+For each clause in `whereQueryObject`, an attribute of `object` is compared to a value (primitive or object.)
+
+## equals (default)
```

Comment:
this is redudant with signle value, below, no?

Reply:
The general description of 'equals' before the single value, right? Yeah.  I would leave the empty title before single value, I think.